### ISSUE TITLE
I can't play any sound.

### ISSUE THREAD

**Issue by soscaster** [rom1v/sndcpy#20]
Removed -Idummy, VLC opened but when Ctrl + M, it always appear the message : 
main error: connection error: Connection refused by peer
main error: connection error: Connection refused by peer
main error: connection error: Connection refused by peer
main error: connection error: Connection refused by peer
main error: connection error: Connection refused by peer
main error: connection error: Connection refused by peer
main error: connection error: Connection refused by peer
main error: connection error: Connection refused by peer

My device is MI A2 Lite - Android 10 Stock ROM.

**Comment by rom1v** [comment]
Did you have the popup on the device and authorized to capture sound, before pressing Enter in the terminal?

**Comment by soscaster** [comment]
> Did you have the popup on the device and authorized to capture sound, before pressing Enter in the terminal?

Yes I have. A huge popup and I authorized it. I can use the scrcpy but not the sndcpy.

**Comment by yulose9** [comment]
> > Did you have the popup on the device and authorized to capture sound, before pressing Enter in the terminal?
> 
> Yes I have. A huge popup and I authorized it. I can use the scrcpy but not the sndcpy.

We do have the same problem I've tried running scrcpy and load the bat file of sndcpy. It did install and app and asked for permission but still didn't play any audio I tried updating my VLC to latest version and still doesn't work

My device is Mi9T Pro running the latest MIUI 12 Global Stable

**Comment by rom1v** [comment]
What app do you use on the device? Some apps do not authorize to capture their audio: https://github.com/rom1v/sndcpy#apps-restrictions

**Comment by yulose9** [comment]
> What app do you use on the device? Some apps do not authorize to capture their audio: https://github.com/rom1v/sndcpy#apps-restrictions

I will try to manifest xml and keep you updated

**Comment by Luukuton** [comment]
Possibly same problem here (Android 10 / OnePlus 7). Tried with Windows 10 Pro 64bit and Arch Linux 5.7.8-arch1-1. 

Connection is succesful and VLC (also tried mpv) does play something but no audio can be heard. I tried playing a video file (mp4) through Chrome, music through AIMP and the main menu of Fate/Grand Order game. No sound with any of these. I tried to record Fate normally just with a recording app, and that worked, so there are no restrictions in play here.

Also, I've tried the latest adb and the one which comes with the Windows binary.

**Comment by rom1v** [comment]
Does it work with the youtube app or VLC for Android?

**Comment by rom1v** [comment]
> I tried to record Fate normally just with a recording app, and that worked

Does this recording app use the Android 10 capture API? Which app is it?

Maybe something to add here:

https://github.com/rom1v/sndcpy/blob/f0995ca4d0002ef288c74c70d4c0eb48354449d5/app/src/main/java/com/rom1v/sndcpy/RecordService.java#L136-L138

?

**Comment by Luukuton** [comment]
> Does it work with the youtube app or VLC for Android?
Yes, I got it working with YouTube. 
> Does this recording app use the Android 10 capture API? Which app is it?
I'm using OxygenOS's internal recording app. Just tried a 3rd party app, AZ Screen Recorder, and it's not picking up any game audio. Probably should've tested that first.

**Comment by Luukuton** [comment]
Possibly add couple example apps to the README which are known to function (eg. YouTube), so people can quickly test that everything works?

Anyway, no problem with sndcpy itself 😅.
